### PR TITLE
DS-104: fixing issue with incorrectly aligned steps in troubleshootin…

### PR DESCRIPTION
**Pull request by chtyler**
…g section

Resolved step ordering problem in one of the troubleshooting admin sections.

## Description
The ordering of the steps was incorrect and need tidying up to ensure that the numerical order flows correctly. 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by grainnejenningsRH — APPROVED** [review]

File: modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc [hunk 1]
```diff
@@ -18,17 +18,8 @@ If you have configured specialized user groups for {productname-short}, the user
...
-.. Log in to {openshift-platform} web console.
---
-endif::[]
-ifdef::upstream[]
-.. Log in to {productname-short} web console.
-endif::[]
-.. Click *User Management* -> *Groups*.
-.. Click the name of your user group, for example, `{oai-user-group}`.
+. In the {openshift-platform} web console, click *User Management* -> *Groups*.
+. Click the name of the default user group.
```
Comment (jstourac) [inline modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc:22]: What is meant by `default user group`? Do we refer to the user group used in the RHOAI product?

Comment (chtyler) [reply modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc:22]: The user groups are explained in this section here: https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2.5/html/managing_users/adding-users_user-mgmt

Customers can configure their own user groups specific to RHOAI. For example, for a normal users (not admins), they can create a group called rhoai users, and add users to that group.

Comment (jstourac) [reply modules/troubleshooting-common-problems-in-jupyter-for-administrators.adoc:22]: Yeah, my point was more about whether it's clear for the reader what we mean by the term `default user group`. But looking also on lines above and it is probably okay from that context. Anyway, it's not a part of this issue.

**Review by jstourac — APPROVED** [review]
Thank you, LGTM.

**Pull request closed without merging**